src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -59,10 +59,23 @@ fun List<OppfolgingstilfelleDag>.groupOppfolgingstilfelleList(): List<Oppfolging
     return oppfolgingstilfelleList
 }
 
-fun List<OppfolgingstilfelleDag>.isArbeidstakerAtTilfelleEnd() =
+fun List<OppfolgingstilfelleDag>.isArbeidstakerAtTilfelleEnd(): Boolean {
+    val last = findLastPriorityOppfolgingstilfelleBit()
+    return if (this.any { it.priorityOppfolgingstilfelleBit?.tagList?.contains(Tag.BEKREFTET) == true } &&
+        this.none { it.priorityOppfolgingstilfelleBit?.tagList?.contains(Tag.SENDT) == true } &&
+        this.none { it.priorityOppfolgingstilfelleBit?.tagList?.contains(Tag.INNTEKTSMELDING) == true } &&
+        last?.tagList?.contains(Tag.NY) == true
+    ) {
+        false
+    } else {
+        last?.isArbeidstakerBit() ?: false
+    }
+}
```

Comment:
Tror ikke jeg har nok innsikt til å si om disse kriteriene er riktige 🤷🏼‍♂️ Får evt diskutere det?

Reply:
Ja, blir fort litt innfløkt dette, men de som er selvstendig næringsdrivende eller liknende bekrefter sykmeldinger (dvs bekrefter i stedet for å sende). Inntektsmelding kommer fra arbeidsgiver, så det indikerer et arbeidsforhold. Sykmelding med tag NY er de som er innsendt, men verken sendt eller bekreftet: Vi bruker disse for å fange opp de som er ikke-digitale. Problemet er å fange opp disse uten at selvstendig næringsdrivende også blir regnet som arbeidstakere.

Reply:
Usikker. Det må i så fall være flere funksjoner på List<OppfolgingstilfelleDag>. Vi får tenke på om det går an å forbedre i en senere PR.